### DISCUSSION TITLE
High heart rate warning

### DISCUSSION THREAD

**Discussion by 88572** [InfiniTimeOrg/InfiniTime#1812] [Ideas]
Hello, I have severe anxiety. Sometimes, when I have a panic attack, my heart rate goes really high (110+ BPM). I would like a way to get a warning when this happens. I could possibly learn c++ for this.

Thanks!

**Comment by borkymcgee** [comment]
As is, the heartrate sensor is only ever even ON when the watch screen is active, making it a poor candidate for detecting momentary states like that. There is a PR in the works for background measurement #1718, perhaps once that is pushed through we could look at implementing something like this, but i kind of fear the implication of InfiniTime being used as a medical-adjacent device in that way.

That being said, if one wanted to implement such a thing, one could potentially look at generating a notification when high heart rates were detected.
